### DISCUSSION TITLE
Full screen behaviour to be more focused on the Biblical text and Dark mode

### DISCUSSION THREAD

**Discussion by augustin-colesnic** [ezra-bible-app/ezra-bible-app#349] [Ideas]
I took my tablet for a 'drive test' today in my local church. :smirk: 
I post here some pictures with the 'feeling' of the full screen mode on Ezra Bible App and 2 more apps.

I think it will benefit the user if we get rid of the buttons and all other elements that clutters the view in full screen mode. It should be only the Biblical text the one to fill the whole screen. Please see attached drawing on the screenshot:

Ahhh and Dark mode will be a great feature, too. I do not know, will the Dark mode be with next release?

![IMG_0104](https://user-images.githubusercontent.com/592416/121773663-0bdcad80-cb86-11eb-96f0-31caacc7885c.jpg)
![IMG_0106](https://user-images.githubusercontent.com/592416/121773667-0f703480-cb86-11eb-8269-c1366a2e6c42.jpg)
![IMG_0107](https://user-images.githubusercontent.com/592416/121773669-10a16180-cb86-11eb-8bfa-b140d6eb594f.jpg)
![IMG_0110](https://user-images.githubusercontent.com/592416/121773670-1139f800-cb86-11eb-97fd-5e3ba293f391.jpg)

**Comment by tobias-klein** [comment]
Cool! 😎
Good to see that you're using the app at church!
There are already options to hide the chapter navigation and the tag/note indicators. We can add an option to hide the majority of the menu. And we could add an option to automatically hide all those things only in fullscreen mode.

**Reply by augustin-colesnic** [reply]
What about the dark mode? Too much light right now 😎

**Reply by tobias-klein** [reply]
It should work on Android!

**Reply by tobias-klein** [reply]
The only thing not implemented is the automatic enabling of dark mode based on the system settings. See #193. We could prioritize this one for the next release!
